### PR TITLE
quincy: doc/rados: update "Pools" material

### PR DESCRIPTION
**Pull request by zdover23**
Remove mentions of default pools (which no longer exist) and improve the syntax of the Pools front matter.

Signed-off-by: Zac Dover <zac.dover@gmail.com>
(cherry picked from commit 956507270465de7d2847f085da22054c42fa3fad)

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/rados/operations/pools.rst [hunk 1]
```diff
@@ -1,38 +1,35 @@
...
-  setting up multiple pools, be careful to set a reasonable number of
-  placement groups for each pool and for the cluster as a whole.  Note that each PG
-  belongs to a specific pool, so when multiple pools use the same OSDs, you must
-  take care that the **sum** of PG replicas per OSD is in the desired PG per OSD
+Pools are logical partitions that are used to store objects.
+
+Pools provide:
+
+- **Resilience**: It is possible to set the number of OSDs that are allowed to
+  fail without any data in the cluster being lost. If your cluster uses
```
Comment (anthonyeleven) [inline doc/rados/operations/pools.rst:9]: s/in the cluster//

File: doc/rados/operations/pools.rst [hunk 2]
```diff
@@ -1,38 +1,35 @@
...
+  replicated pools, the number of OSDs that can fail without data loss is the
+  number of replicas of an object. For example: a typical configuration stores
```
Comment (anthonyeleven) [inline doc/rados/operations/pools.rst:11]: s/of an object//

File: doc/rados/operations/pools.rst [hunk 3]
```diff
@@ -1,38 +1,35 @@
...
+  pools <../erasure-code>`_, resilience is defined as the number of coding
+  chunks (for example, ``m = 2`` in the **erasure code profile**).
+
+- **Placement Groups**: You can set the number of placement groups for the
+  pool. A typical configuration targets approximately 100 placement groups per
+  OSD, providing optimal balancing without consuming many computing resources.
+  When setting up multiple pools, be careful to set a reasonable number of
+  placement groups for each pool and for the cluster as a whole. Note that each
+  PG belongs to a specific pool: when multiple pools use the same OSDs, make
+  sure that the **sum** of PG replicas per OSD is in the desired PG per OSD
```
Comment (anthonyeleven) [inline doc/rados/operations/pools.rst:23]: Reference `pgcalc` and/or `ceph osd df`?

**Review by anthonyeleven — APPROVED** [review]
Several non-blocking suggestions.

File: doc/rados/operations/pools.rst (hunk 3 above)
Comment (zdover23) [reply doc/rados/operations/pools.rst:23]: Note to the future: "ceph osd df" doesn't have a lot of documentation as of 14 Nov 2022.

File: doc/rados/operations/pools.rst (hunk 1 above)
Comment (zdover23) [reply doc/rados/operations/pools.rst:9]: See https://github.com/ceph/ceph/pull/48866.

File: doc/rados/operations/pools.rst (hunk 2 above)
Comment (zdover23) [reply doc/rados/operations/pools.rst:11]: See https://github.com/ceph/ceph/pull/48866.

File: doc/rados/operations/pools.rst (hunk 3 above)
Comment (zdover23) [reply doc/rados/operations/pools.rst:23]: See https://github.com/ceph/ceph/pull/48866.

**Pull request merged**